### PR TITLE
fix: ActionBar should not appear in create/edit route resultview

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [x] Improve performance

- Related issues
fix issue: #418 
___
### Bugfix
- Description

- How to fix?

condition to show acttionbar 
___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: src/pages/Route/Create.tsx [hunk 1]
```diff
@@ -270,7 +270,11 @@ const Page: React.FC<Props> = (props) => {
           {renderStep()}
         </Card>
       </PageHeaderWrapper>
-      <ActionBar step={step} lastStep={redirect ? 2 : 4} onChange={onStepChange} withResultView />
+      {step !== 5 ? (
+        <ActionBar step={step} lastStep={redirect ? 2 : 4} onChange={onStepChange} withResultView />
```
Comment (juzhiyuan) [inline src/pages/Route/Create.tsx:274]: Why not update Action directly?

Comment (liuxiran) [reply src/pages/Route/Create.tsx]: > Good point but why not update the ActionBar component directly?


thanks for your proposal, update actionbar is more elegant and versatile. I've learned.

File: src/components/ActionBar/ActionBar.tsx [hunk 2]
```diff
@@ -44,6 +44,11 @@ const ActionBar: React.FC<Props> = ({ step, lastStep, onChange, withResultView }
     return null;
   }
 
+  // resultView should not show actionbar
+  if (step > lastStep && withResultView) {
```
Comment (juzhiyuan) [inline src/components/ActionBar/ActionBar.tsx:48]: `>=` I think? 🤔

Comment (liuxiran) [reply src/components/ActionBar/ActionBar.tsx:48]: > `>=` I think? thinking

**Pull request merged**